get rid of unncessary fx wrap in regrouping

ZhengkaiZ opened the pull request:
Summary: [torchrec] get rid of unncessary fx wrap in regrouping

Differential Revision: D72884977

cp2923 reviewed (approved):
Double check, is there a test plan? Or existing UTs are sufficient

ZhengkaiZ committed:
get rid of unncessary fx wrap in regrouping (#2882)

Summary:

[torchrec] get rid of unncessary fx wrap in regrouping

Reviewed By: jingsh, taylorKempWork, FulinHuang

Differential Revision: D72884977
